src/styles/card.scss

```diff
@@ -0,0 +1,55 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  @include transition-time(all);
```

Comment:
Specify to what properties should transition apply, avoid `all`
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x